src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/varsel/VarselbrevUtil.kt

```diff
@@ -120,6 +130,80 @@ class VarselbrevUtil(private val eksterneDataForBrevService: EksterneDataForBrev
                                    varsletBeløp = varsel?.varselbeløp)
     }
 
+    private fun sammenstillInfoFraSimuleringForVedlegg(varselbrevsdokument: Varselbrevsdokument,
+                                                       eksternBehandlingId: String,
+                                                       varsletTotalbeløp: Long): Vedleggsdata {
+
+        val feilutbetalingerFraSimulering =
```

Comment:
Den bør kalles kun for EF. Hvordan stopper du det for BA?

Reply:
I funksjonen lenger ned som kaller denne.